src/odemis/acq/stream/_sync.py

```diff
@@ -774,7 +780,11 @@ def _assembleTiles(self, rep, data_list):
 
         # start with the metadata from the first point
         md = data_list[0].metadata.copy()
-        center, pxs = self._get_center_pxs(rep, (T, S), data_list[0])
+        center_tl = md[MD_POS]
+        pxs = md[MD_PIXEL_SIZE]
+        shape_tl = (S, T)
```

Comment:
```suggestion
        shape_tl = (T, S)
```
previously it was `(T, S)`, was it incorrect?

Reply:
That's indeed correct. That was one of my changes in the refactoring of the API: all arguments are sorted X, Y. (No more mix of "numpy order" and "conventional order").